src/premiscale/metrics/__init__.py

```diff
@@ -120,9 +120,74 @@ def __call__(self) -> None:
 
         self.stateConnection = build_state_connection(self.config)
         self.stateConnection.open()
-
+        self.stateConnection.initialize()
+        self._initialize_host()
         self._collectMetrics()
 
+    def _initialize_host(self, host: Host | None = None) -> None:
```

Comment:
Why don't I just create a method on the Host object itself and simplify the hell out of this.